infrastructure/ami/scripts/install-huggingface-libraries.sh

```diff
@@ -9,24 +9,22 @@ echo "TRANSFORMERS_VERSION: $TRANSFORMERS_VERSION"
 echo "OPTIMUM_VERSION: $OPTIMUM_VERSION"
 
 pip install --upgrade --no-cache-dir \
-    "transformers[sklearn,sentencepiece,vision]==$TRANSFORMERS_VERSION" \
-    "datasets==2.16.1" \
-    "accelerate==0.23.0" \
-    "diffusers==0.25.0" \
     "evaluate==0.4.1" \
     "requests==2.31.0" \
     "notebook==7.0.6" \
     "markupsafe==2.1.1" \
     "jinja2==3.1.2" \
     "attrs==23.1.0"
 
+# Temporary fix for the issue: https://github.com/huggingface/optimum-neuron/issues/142
+pip install -U optimum
```

Comment:
lets pin the version?

Reply:
I think from now on `optimum-neuron` will be always compatible with the latest optimum since the installation of the latest optimum-neuron via optimum's neuronx extra won't work otherwise